source/api/commands/as.md

```diff
@@ -70,6 +91,47 @@ cy.wait('@userPut')
 
 Some strings are not allowed as alias names since they are reserved words in Cypress. These words include: `test`, `runnable`, `timeout`, `slow`, `skip`, and `inspect`.
 
+## `as` is asynchronous
```

Comment:
A wee section that gives more explicit examples to address #722.

It's only targeting `fixture()` aliases but I thought that adding a ton of examples for aliasing `route()` and DOM elements might be overload.